### DISCUSSION TITLE
What if for a given month there is no data for out_of_network array due to 20 claim rule, how should this be handled?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#79] [Q&A] [answered]
There is a potential scenario for a small reporting plan_name/plan_id that may not have the 20 unique claim scenario for the same TIN, same item/service to report some months for their out of network providers, as they have a very small percentage of claims being out of network.    The schema has the out_of_network array as required---so how should this scenario be handled?  My assumption is that they would need to have a file for the month--but schema doesn't seem to support this scenario.     We can hope they have at least one scenario that meets the 20 unique claim scenario--but if this criteria is not met--then how should this be handled?

**Comment by Ginger228** [comment]
This is a scenario I have been investigating.  If the members are not seeing out of networks providers or the plan is a HMO.  The Allowed Amount would not exist.  To avoid confusion I felt it best to not generate an empty file.

**Reply by tdfow** [reply]
Definitely need direction on what is expected for this scenario, so there is consistency in handling and expectations.

**Comment by shaselton-usds** [comment, marked as answer]
@tdfow Just following up on this. Did you have a chance to see what an "empty" allowed amounts file would look like? You can find an example here: https://github.com/CMSgov/price-transparency-guide/blob/59a3adc4f9d1ef6f86a306ea8714112db1805185/examples/allowed-amounts/allowed-amounts-single-plan-empty-sample.json

**Reply by tdfow** [reply]
@shaselton   Thank you.

**Reply by lynnlgeha** [reply]
I noticed the link for the example has been provided multiple times in this thread; however, the link returns an error.  Will the example be updated or is there another link for the example file to support this question?

**Reply by tdfow** [reply]
https://github.com/CMSgov/price-transparency-guide/tree/master/examples/allowed-amounts

**Reply by tdfow** [reply]
@lynnlgeha  since CMS may alter the links---it is always best to start here---and navigate to the examples :)  https://github.com/CMSgov/price-transparency-guide
